modules/RTC/JitsiLocalTrack.js

```diff
@@ -368,7 +370,7 @@ export default class JitsiLocalTrack extends JitsiTrack {
         const logMuteInfo = () => logger.info(`Mute ${this}: ${muted}`);
 
         if (this.isAudioTrack()
-                || this.videoType === VideoType.DESKTOP
+                || (this.videoType === VideoType.DESKTOP && !FeatureFlags.isMultiStreamSupportEnabled())
```

Comment:
can you explain how muting a desktop track will work? does it remove the actual stream from the conference? why is it not enough to just use the "enabled" flag like for audio? is the AV moderation the main use case for being able to mute a desktop track?

Reply:
Can you add a comment which explains why mute in multi stream works different than in regular mode? Also mention that this behavior is used in jitsi-meet to avoid extra source-remove/source-add?